test/cljs/status_im/test/models/browser_history.cljs

```diff
@@ -1,117 +0,0 @@
-(ns status-im.test.models.browser-history
```

Comment:
No more tests?

Reply:
yeah no model, and now it's super simple code, nothing to test

Reply:
@flexsurfer there's much to test `browser-back` `browser-forward` `update-browser-history-tx`, if we don't start adding tests to the codebase (instead of removing them :) ) it makes it harder for everyone to modify and improve it